status/types.proto

```diff
@@ -87,3 +90,12 @@ enum CommonFail {
     //  unsigned integer).
     WRONG_MAGIC_NUMBER = 1;
 }
+
+// Section of failed statuses for object-related operations.
+enum Object {
+    // [**2048**] Operation rejected by the object lock.
+    LOCKED = 0;
+
+    // [**2049**] Locking of the irregular object rejected.
+    IRREGULAR_LOCK = 1;
```

Comment:
`WRONG_OBJECT_TYPE`? The "irregular" may be misinterpreted.

Reply:
Then we need some details since `WRONG_OBJECT_TYPE` is too generic.

Reply:
`WRONG_LOCKED_OBJECT_TYPE`?

Reply:
Renamed to `LOCK_NON_REGULAR_OBJECT`